src/uwtools/config/jinja2.py

```diff
@@ -204,6 +201,62 @@ def render(
 # Private functions
 
 
+def _deref_convert(val: TaggedString) -> _ConfigVal:
+    """
+    Convert a string tagged with an explicit type.
+
+    If conversion cannot be performed (e.g. the value was tagged as an int but int() is not a value
+    that can be represented as an int, the original value will be returned unchanged.
+
+    :param val: A scalar value tagged with an explicit type.
+    :return: The value translated to the specified type.
+    """
+    converted: _ConfigVal = val  # fall-back value
+    _deref_debug("Converting", val.value)
+    try:
+        converted = val.convert()
+        _deref_debug("Converted", val.value)
```

Comment:
Does it make sense to use `converted` in the debug message here? `val.value` was reported int he previous debug message.

Reply:
Yes, I think you're right. Thanks.